src/components/ui/File.css

```diff
@@ -0,0 +1,84 @@
+.message .message-text > .image-file {
+    position: relative;
+    width: 50%;
+}
+
+/*.message .message-text > .image-file:hover > img {
```

Comment:
Could you remove this? Not a huge fan of commented code in the project :wink:

Reply:
Shame on me ahhahaha !